src/app/ui/cpvareditor/process.go

```diff
@@ -254,6 +259,10 @@ func (v *VarEditor) showVarInput(varName string) {
 		}).Tooltip(initialValue).Style(style.ButtonFrame{})
 	}
 
+	if readonly {
+		imgui.BeginDisabled()
+	}
+
```

Comment:
Lets move `BeginDisabled`/`EndDisabled` from the the method upper, so the variable name will be disabled too.